### PR TITLE
Add cardano RPC

### PR DESCRIPTION
**Pull request by neuodev**
[Task Link](https://app.asana.com/0/1200458692294040/1200567965519644/f)

### CODE REVIEW

File: packages/yoroi-ergo-connector/src/inject.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 // sets up RPC communication with the connector + access check/request functions
 const initialInject = `
-var ergoConnectRequests = [];
+var connectRequests = [];
 
+var cardanoConnectRequests = [];
```
Comment (vantuz-subhuman) [inline packages/yoroi-ergo-connector/src/inject.js:5]: @AhmedIbrahim336 , I think you don't need this `cardanoConnectRequests` at all

Comment (vantuz-subhuman) [reply packages/yoroi-ergo-connector/src/inject.js:5]: Everything else is good 👌

Comment (zuzunker) [inline packages/yoroi-ergo-connector/src/inject.js]:
```suggestion
```

**Review by zuzunker — APPROVED** [review]

**Comment by cvetelinandreevdreamix** [comment]
Hey @vsubhuman,

We have an existing iOS app which we want to integrate with a Cardano wallet. Basically we want to open a wallet app to sign a transaction we've prepared in advance. We'll use a testnet initially.

Is that possible now?

Will be there nightly builds for the iOS app we can test (TestFlight)?

Thank you!

**Pull request merged**